app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -14,6 +14,7 @@ import getDateFormat from '../../actions/dateFormat';
 import activeOrderAction from '../../actions/activeOrderAction';
 import { fetchPatientRecord, fetchPatientNote } from '../../actions/patient';
 import { setSelectedOrder } from '../../actions/orderAction';
+import toggleDraftLabOrderUrgency from '../../actions/draftActions';
```

Comment:
`name of import` should match `export-default`

Reply:
The file `draftAction` is still going to contain a lot of actions that belong to the draft some of which are to be implemented in [this](https://issues.openmrs.org/browse/OEUI-236) and [this](https://issues.openmrs.org/browse/OEUI-239) tickets.
That is the reason why I named the file `draftActions`, once other actions are added to the file, the `export-default` won't be needed anymore.

Reply:
If there was a rule to lint this, you know you'll probably have to fix it. I understand your point but the code written at any point in time should adhere to conventions used at that point in time.

Reply:
since this PR and the related PR's will be merges soonest, you can leave it as it is but just bear in mind that this is not acceptable under normal circumstances.